tests/unittest_for_iterator.py

```diff
@@ -0,0 +1,23 @@
+import unittest
+import os
+import sys
+import inspect
+current_dir = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
+parent_dir = os.path.dirname(current_dir)
+sys.path.insert(0, parent_dir)
+import range_iterator
+from range_iterator import *
+
+
+class TestIterator(unittest.TestCase):
+
+    def test_MyRange(self):
```

Comment:
Is this test enough?

Reply:
Not enough, it's just the simpliest test to understand how unit-tests work